packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -1,69 +1,73 @@
-import React from 'react';
-import styles from '@patternfly/react-styles/css/components/ContextSelector/context-selector';
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/ContextSelector/context-selector'
 import { css } from '@patternfly/react-styles';
-import PropTypes from 'prop-types';
-import FocusTrap from 'focus-trap-react';
 import { SearchIcon } from '@patternfly/react-icons';
-import ContextSelectorToggle from './ContextSelectorToggle';
-import ContextSelectorMenuList from './ContextSelectorMenuList';
+import { ContextSelectorToggle } from './ContextSelectorToggle';
+import { ContextSelectorMenuList } from './ContextSelectorMenuList';
 import { ContextSelectorContext } from './contextSelectorConstants';
 import { Button, ButtonVariant } from '../Button';
 import { TextInput } from '../TextInput';
 import { InputGroup } from '../InputGroup';
 import { KEY_CODES } from '../../helpers/constants';
 
+// Can't use ES6 imports :(
+// The types for it are also wrong, we should probably ditch this dependency.
+// tslint:disable-next-line
+const FocusTrap: any = require('focus-trap-react');
+
 // seed for the aria-labelledby ID
 let currentId = 0;
 const newId = currentId++;
 
-const propTypes = {
+export interface ContextSelectorProps {
   /** content rendered inside the Context Selector */
-  children: PropTypes.node,
+  children?: React.ReactNode;
   /** Classes applied to root element of Context Selector */
-  className: PropTypes.string,
+  className?: string;
   /** Flag to indicate if Context Selector is opened */
-  isOpen: PropTypes.bool,
+  isOpen?: boolean;
```

Comment:
Isn't this required?

Reply:
I think they were already optional in the js. I thought they were only required if the `.isRequired` is on the end for the js proptypes?